dashboard/src/presenters/toAreaChart.ts

```diff
@@ -25,27 +26,66 @@ type ToAreaChartProps<K extends string> = DataToAreaChartProps<K> & {
   };
 };
 
+function normalizeIterationRange(dateRange: { start: Date; end: Date }, granularity: GranularityRangeValues) {
+  if (granularity === 'day') {
+    const startUTC = new Date(
+      Date.UTC(
+        dateRange.start.getUTCFullYear(),
+        dateRange.start.getUTCMonth(),
+        dateRange.start.getUTCDate(),
+        0,
+        0,
+        0,
+        0,
+      ),
```

Comment:
I think this can be simplified to:
`utcDay(dateRange.start)` or `new Date(utcDay(dateRange.start))`